src/utils/copy-file.js

```diff
@@ -16,6 +16,12 @@ export default function copyFile(sourcePath, destPath) {
   try {
     symlinkOrCopy.sync(sourcePath, destPath);
   } catch(e) {
+    // TODO: change mr-dep-walk API to expose found vs missing (aka external) deps
```

Comment:
created https://github.com/stefanpenner/mr-dep-walk/issues/4 to work on removing this